static/homepage/index.html

```diff
@@ -120,7 +120,7 @@ <h4 class="hero-tagline">Save, highlight, send
               <a class="btn btn-primary btn-lg main-get-started installer" href="https://chrome.google.com/webstore/detail/pageshot/flhjebpmjblnihhioankmibfdkdiaahi">Install</a>
             </p>
             <p class="if-not-firefox-or-chrome">
-              <a class="btn btn-primary btn-lg" href="https://www.mozilla.org/en-US/firefox/new/">To use PageShot: Install Firefox!</a><br>
```

Comment:
Removed the `/en-US/` here since mozilla.org will redirect for us, and give the user their preferred/best locale (whenever possible).